Makefile

```diff
@@ -0,0 +1,16 @@
+.PHONY: help
+help: ## Displays this list of targets with descriptions
+	@echo "The following commands are available:\n"
+	@grep -E '^[a-zA-Z0-9_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[32m%-30s\033[0m %s\n", $$1, $$2}'
+
+
+.PHONY: docs
+docs: ## Generate projects docs (from "Documentation" directory)
+	mkdir -p Documentation-GENERATED-temp
```

Comment:
does this directory make sense to be added to .gitignore?

Reply:
I removed the makefile for now